src/main/docs/guide/quickStart.adoc

```diff
@@ -1,10 +1,10 @@
 Add the following dependency to your Micronaut application:
 
-dependency:io.micronaut.reactor:micronaut-reactor[scope="compile", version="{version}"]
+dependency:micronaut-reactor[scope="compile", groupId="io.micronaut.reactor", version="{version}"]
 
 To use the HTTP client add the following dependency:
 
-dependency:io.micronaut.reactor:micronaut-reactor-http-client[scope="compile", version="{version}"]
+dependency:micronaut-reactor-http-client[scope="compile", groupId="io.micronaut.reactor", version="{version}"]
```

Comment:
this generates 

```
implementation("io.micronaut.reactor:micronaut-reactor-http-client:2.2.1-SNAPSHOT")
```

I think what we should have is: 

```
implementation("io.micronaut.reactor:micronaut-reactor-http-client")
```

which we could achieve with: 

```
dependency:micronaut-reactor-http-client[groupId="io.micronaut.reactor"]
```

Reply:
@sdelamo Good catch...  I manually updated the docs https://micronaut-projects.github.io/micronaut-reactor/latest/guide/ which shows 2.2.0 (as I was on the released commit when I generated them).  Do you think we need to manually publish again?  Or is this ok to wait until next release?

Reply:
I think it is ok to wait.